pyproject.toml

```diff
@@ -130,3 +131,23 @@ ignore = [
 
 [tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401"]
+
+[tool.cibuildwheel]
+build-frontend="build"
+environment = {USE_BAZEL_VERSION = "6.5.0"}
+skip = ["*musllinux*", "pp*"]
+# skip = ["cp312-*", "cp313-*", "*musllinux*", "pp*"]
+
+test-extras = ["test"]
+before-test="rm {project}/bazel-*"
+test-command="pytest {project}"
+
+[tool.cibuildwheel.linux]
+#manylinux-x86_64-image = "manylinux_2_28"
+manylinux-x86_64-image = "manylinux2014"
+archs=["x86_64"]
+before-build = "yum install -y npm && npm install -g @bazel/bazelisk"
+
+
+[tool.cibuildwheel.macos]
+archs = ["arm64"]
```

Comment:
How does this work? As far as I can tell on MacOS there's no bazelisk installation.

Reply:
bazelisk is already installed on the [macos github action runners](https://github.com/actions/runner-images/blob/main/images/macos/macos-15-Readme.md#utilities), so no need to reinstall.